packages/eddsa-pcd/src/index.ts

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="./util/declarations/circomlibjs.d.ts" />
```

Comment:
For future reference, an equivalent to this line is really important if you want apps/other packages to be able to import a package which provides custom types for an npm module that it's importing.